Sources/SwiftInspectorVisitors/FileVisitor.swift

```diff
@@ -100,6 +101,7 @@ public final class FileVisitor: SyntaxVisitor {
     fileInfo.appendStructs(declarationVisitor.structs)
     fileInfo.appendClasses(declarationVisitor.classes)
     fileInfo.appendEnums(declarationVisitor.enums)
+    fileInfo.appendTypealiases(declarationVisitor.typealiases)
```

Comment:
Is this finding typealiases in a top-level declaration?

Reply:
This line is propagating typealiases that are defined within a nestable type (i.e. defined within a `class`, `enum`, or `struct`).